README.md

```diff
@@ -185,6 +185,8 @@ This will create a new file on S3 with this signature:
 s3://{METRICS_BUCKET}/aggregated/{module}/{benchmark_name}/results_automlbenchmark_{constraint}_{benchmark_name}.csv
 ```
 
+Currently, aggregation is also supported for multimodal benchmark results without the `--constratint` option.
+
```

Comment:
Do you think we will ever have the idea of constraints for multimodal?

Reply:
Yes I'm planning to add constraint to multimodal, but ideally I want to have it look like the same for multimodal and tabular. So I'm still thinking about the design.